kratos/geometries/prism_3d_15.h

```diff
@@ -727,10 +727,10 @@ template<class TPointType> class Prism3D15 : public Geometry<TPointType>
         Matrix ShapeFunctionsGradientInIntegrationPoint =
             shape_functions_gradients( IntegrationPointIndex );
         //values of shape functions in integration points
-        vector<double> ShapeFunctionsValuesInIntegrationPoint = ZeroVector( 15 );
+        //DenseVector<double> ShapeFunctionsValuesInIntegrationPoint = ZeroVector( 15 );
```

Comment:
why did u comment this?

Reply:
I'm not sure but seems that is not used and gives warning

Reply:
ok, if it compiles without fine to me.